Project.toml

```diff
@@ -1,12 +1,7 @@
 name = "ClimaCoupler"
 uuid = "4ade58fe-a8da-486c-bd89-46df092ec0c7"
-authors = ["jbmackay <jbmackay@caltech.edu>"]
+authors = ["CliMA Contributors <clima-software@caltech.edu>"]
 version = "0.1.0"
 
 [deps]
-ClimaCommsMPI = "5f86816e-8b66-43b2-912e-75384f99de49"
-Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
-OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
```

Comment:
Similarly, only packages that are required for code in `src` should live in the Project.toml. Packages like SciMLBase and OrdinaryDiffEq are just needed for running models and can live in the Project.toml files in `experiments/`

Reply:
Sounds good!